shell/platform/android/android_surface_gl.cc

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include <utility>
 #include "flutter/shell/platform/android/android_surface_gl.h"
```

Comment:
Clang format applies the header ordering rules automatically now. But it decides to stop if we introduce a space anywhere. This way, we can just plop includes anywhere we want and let Clang format do its thing correctly.

I didn't notice anything in https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes about spaces but I am happy to add the same if this is an unwritten rule.